docs/sources/administration/service-accounts/index.md

```diff
@@ -153,3 +153,59 @@ You can assign on of the following permissions to a specific user or a team:
 1. In the **Permissions** section at the bottom, click **Add permission**.
 1. Choose **User** in the dropdown and select your desired user.
 1. Choose **View**, **Edit** or **Admin** role in the dropdown and click **Save**.
+
+## Debug the permissions of a service account token
+
+Sometimes, for debugging purposes, you might need to know which RBAC permissions are attached to a service account token.
+This section will give you an overview of how to do it.
```

Comment:
```suggestion
This section explains how to learn which RBAC permissions are attached to a service account token.
This can help you diagnose permissions-related issues with token authorization.
```

Reply:
perfect!